### ISSUE TITLE
Discord rat 3.0

### ISSUE THREAD

**Issue by Lbozoisback** [moom825/Give-me-idea#26]
This but with some thing from 2.0 like password grabberi want all of this in the 2..0 or 3.0 because 1.0 aka discord rat doesnt work for me :Available commands are :
--> !message = Show a message box displaying your text / Syntax  = "!message example"
--> !shell = Execute a shell command /Syntax  = "!shell whoami"
--> !webcampic = Take a picture from the webcam
--> !windowstart = Start logging current user window (logging is shown in the bot activity)
--> !windowstop = Stop logging current user window 
--> !voice = Make a voice say outloud a custom sentence / Syntax = "!voice test"
--> !admincheck = Check if program has admin privileges
--> !sysinfo = Gives info about infected computer
--> !history = Get chrome browser history
--> !download = Download a file from infected computer
--> !upload = Upload file to infected computer / Syntax = "!upload file.png" (with attachment)
--> !cd = Changes directory
--> !delete = deletes a file / Syntax = "!delete /path to/the/file.txt"
--> !write = Type your desired sentence on computer / Type "enter" to press the enter button on the computer
--> !wallpaper = Change infected computer wallpaper / Syntax = "!wallpaper" (with attachment)
--> !clipboard = Retrieve infected computer clipboard content
--> !geolocate = Geolocate computer using latitude and longitude of the ip adress with google map / Warning : Geolocating IP adresses is not very precise
--> !startkeylogger = Starts a keylogger
--> !stopkeylogger = Stops keylogger
--> !dumpkeylogger = Dumps the keylog
--> !volumemax = Put volume to max
--> !volumezero = Put volume at 0
--> !idletime = Get the idle time of user's on target computer
--> !blockinput = Blocks user's keyboard and mouse / Warning : Admin rights are required
--> !unblockinput = Unblocks user's keyboard and mouse / Warning : Admin rights are required
--> !screenshot = Get the screenshot of the user's current screen
--> !exit = Exit program
--> !kill = Kill a session or all sessions / Syntax = "!kill session-3" or "!kill all"
--> !uacbypass = attempt to bypass uac to gain admin by using fod helper
--> !passwords = grab all chrome passwords
--> !streamwebcam = streams webcam by sending multiple pictures
--> !stopwebcam = stop webcam stream
--> !getdiscordinfo = get discord token,email,phone number,etc
--> !streamscreen = stream screen by sending multiple pictures
--> !stopscreen = stop screen stream
--> !shutdown = shutdown computer
--> !restart = restart computer
--> !logoff = log off current user
--> !bluescreen = BlueScreen PC
--> !displaydir = display all items in current dir
--> !currentdir = display the current dir
--> !dateandtime = display system date and time
--> !prockill = kill a process by name / syntax = "!kill process.exe"
--> !recscreen = record screen for certain amount of time / syntax = "!recscreen 10"
--> !reccam = record camera for certain amount of time / syntax = "!reccam 10"
--> !recaudio = record audio for certain amount of time / syntax = "!recaudio 10"
--> !disableantivirus = permanently disable windows defender(requires admin)
--> !disablefirewall = disable windows firewall (requires admin)
--> !audio = play a audio file on the target computer(.wav only) / Syntax = "!audio" (with attachment)
--> !selfdestruct = delete all traces that this program was on the target PC
--> !windowspass = attempt to phish password by poping up a password dialog
--> !displayoff = turn off the monitor(Admin rights are required)
--> !displayon = turn on the monitors(Admin rights are required)
--> !hide = hide the file by changing the attribute to hidden
--> !unhide = unhide the file the removing the attribute to make it unhidden
--> !ejectcd = eject the cd drive on computer
--> !retractcd = retract the cd drive on the computer
--> !critproc = make program a critical process. meaning if its closed the computer will bluescreen(Admin rights are required)
--> !uncritproc = if the process is a critical process it will no longer be a critical process meaning it can be closed without bluescreening(Admin rights are required)
--> !website = open a website on the infected computer / syntax = "!website google.com" or "!website www.google.com"
--> !distaskmgr = disable task manager(Admin rights are required)
--> !enbtaskmgr = enable task manager(if disabled)(Admin rights are required)
--> !getwifipass = get all the wifi passwords on the current device(Admin rights are required)
--> !startup = add file to startup(when computer go on this file starts)(Admin rights are required)

**Comment by InfernoTsugikuni** [comment]
yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there

**Comment by solaritygh** [comment]
> yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there

Yeah but I mean, you'd have to make a server on the web for that, and not run the bot through the user computer, which would mean a full rewrite.

**Comment by InfernoTsugikuni** [comment]
> > yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there
> 
> Yeah but I mean, you'd have to make a server on the web for that, and not run the bot through the user computer, which would mean a full rewrite.

true but this is 3.0 so why not pull a LummaC2

**Comment by Lbozoisback** [comment]
Ye and make it so U can change desciption and stuff make it less obv for the rat

**Comment by Lbozoisback** [comment]
OH YE BTW USE CONFUSEREX to obfuscate Than use resource hacker to change desc and filename andd boom

**Comment by Lbozoisback** [comment]
> yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there

But u know that connections are a thing to

**Comment by InfernoTsugikuni** [comment]
> > yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there
> 
> But u know that connections are a thing to

i mean like i dont care, as long as the site is password protected so that they cant ruin your whole operation like with a discord bot token

**Comment by solaritygh** [comment]
> > > yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there
> > 
> > 
> > But u know that connections are a thing to
> 
> i mean like i dont care, as long as the site is password protected so that they cant ruin your whole operation like with a discord bot token

bro that's not possible

**Comment by InfernoTsugikuni** [comment]
> > > > yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there
> > > 
> > > 
> > > But u know that connections are a thing to
> > 
> > 
> > i mean like i dont care, as long as the site is password protected so that they cant ruin your whole operation like with a discord bot token
> 
> bro that's not possible

true im stupid

**Comment by InfernoTsugikuni** [comment]
> > > > > yeah and make it not be so trash since you can get the bot token thru tria.ge when you run the file thru there
> > > > 
> > > > 
> > > > But u know that connections are a thing to
> > > 
> > > 
> > > i mean like i dont care, as long as the site is password protected so that they cant ruin your whole operation like with a discord bot token
> > 
> > 
> > bro that's not possible
> 
> true im stupid

since that would be discordrat but a normal rat and he has xeno rat

**Comment by Lbozoisback** [comment]
Connections are a thing to get the token

**Comment by Lbozoisback** [comment]
I GOT SERVER FUCKED TROUGH CONNECTION I OBFUSCATED MY SHIT

**Comment by Lbozoisback** [comment]
ANd make it like u can add Mb to it junk code

**Comment by solaritygh** [comment]
> I GOT SERVER FUCKED TROUGH CONNECTION I OBFUSCATED MY SHIT

don't rat people
